<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E60D1;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="twoPlayer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/ui/Screen_Shot_2022-05-07_at_9.17.15_PM.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PingFang HK</family>
      <pointsize>50</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow</string>
    </property>
    <property name="text">
     <string>PAC-MAN</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>260</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgorund-color:rgb(0, 243, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onePlayer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/ui/Screen Shot 2022-05-07 at 9.00.44 PM.png) 0 0 0 0 stretch stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="levelOne">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Level 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="levelTwo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="levelThree">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Level 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takeBreak">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Take a Break?</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/ui/Screen_Shot_2022-05-07_at_9.28.44_PM.png)</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>twoPlayer</zorder>
   <zorder>label</zorder>
   <zorder>onePlayer</zorder>
   <zorder>levelOne</zorder>
   <zorder>levelTwo</zorder>
   <zorder>levelThree</zorder>
   <zorder>takeBreak</zorder>
   <zorder>graphicsView_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
